app/src/main/java/de/westnordost/streetcomplete/tangram/MapFragment.java

```diff
@@ -349,6 +352,23 @@ private void updateAccuracy()
 							TextUtils.join(",",directionMarkerSize) +
 							"], order: 2000, collide: false, flat: true, angle: " + r + " }");
 		}
+
```

Comment:
The direction marker should not be visible when the rotate-view-to-device-orientation mode is on because it should always point exactly forward.